code/modules/research/techweb/all_nodes.dm

```diff
@@ -520,6 +520,15 @@
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
 	export_price = 5000
 
+/datum/techweb_node/camera_theory
+	id = "cam_theory"
+	display_name = "Camera Theory"
+	description = "NT is watching you..."
+	design_ids = list("minercam", "bodycam")
+	prereq_ids = list("basic_mining","sec_basic")
```

Comment:
No idea if this is a good place for this. It's pretty early in the round. I could lock it behind more prereqs but I don't know. Could also add it to another node but there is no "optics" or "camera" research node that already exists